### DISCUSSION TITLE
AIO Main PCB Pinout compatible to Darthclouds SMD PCBs?

### DISCUSSION THREAD

**Discussion by stefanpausch** [pmgducati/Blue-Retro-AIO-Units#52] [Q&A] [answered]
Can i build the AIO Main PCB (long version) and use it in darthclouds SMD PCBs? ( i know the female/male connectors have to be swapped). Are both projects pin compatible?

Sadly i cannot read the PCB/Schemata Files in Kicad (i don't have the Eagle Software)

**Comment by pmgducati** [comment, marked as answer]
They should be compatible, They are all designed off darthcloud's schematics.   I swapped the female/male connectors as I want only one male connecter where pins can be damaged vs the original design where each system risks that chance.  But there is no functionality difference having the Male/Female on either side as long as your set is consistent.

**Reply by stefanpausch** [reply]
Thanks for the clarification - Will test it when all parts arrive (Within the next two weeks) and update with a comment :)
